sql/sql_alter_instance.cc

```diff
@@ -269,6 +269,12 @@ bool Innodb_redo_log::execute() {
     return true;
   }
 
```

Comment:
Please add a comment that this lock is Percona's LOCK TABLES FOR BACKUP.

Reply:
Commend added.